modules/development/debug-tools.nix

```diff
@@ -37,6 +37,9 @@ in
           # Performance testing
           speedtest-cli
           iperf
+          perf-tools
+          linuxPackages.perf
```

Comment:
Off-topic: I find bcc's profile script a bit easier to use ` sudo profile -f -F 99 10 > /tmp/folded-graph` and the result can be looked at in https://www.speedscope.app/